src/Engines/nativeEngine.ts

```diff
@@ -2225,6 +2226,22 @@ export class NativeEngine extends Engine {
         // TODO
     }
 
+    /**
+     * Create a canvas
+     * @param width width
+     * @param height height
+     * @return ICanvas interface
+     */
+    public createCanvas(width: number, height: number) : ICanvas {
+        if (!_native.NativeCanvas) {

```

Comment:
should the native engine expose static functions for loading ttfs? where will that logic live?

Reply:
No strong opinion for that yet.